app/views/tweets/show.erb

```diff
@@ -0,0 +1,21 @@
+
+<h1>Welcome, <%=@user.username%></h1>
+
+<h3>Your Fwitter digest:</h3>
+
+<%Tweet.all.reverse.each do |tweet|%>
+  <%user = User.find_by_id(tweet.user_id)%>
+  <a href="/users/<%=user.slug%>"><%=user.username%></a>: <%=tweet.content%> <br>
+  <%end%>
+
+<h3>Edit your Tweets:</a></h3>
+<%x=1%>
+<%@user.tweets.each do |tweet|%>
```

Comment:
Again, we should leave database logic out of the view and set this to an instance variable in the TweetsController

`@user_tweets = current_user.tweets`